bundle/src/main/java/com/adobe/acs/commons/mcp/impl/processes/asset/AssetIngestor.java

```diff
@@ -293,7 +295,9 @@ private void createAsset(Source source, String assetPath, ResourceResolver r, bo
                 }
                 Asset asset = assetManager.createAsset(assetPath, source.getStream(), type, false);
 
-                if (asset != null) {
+                if (asset == null) {
+                    Logger.getLogger(AssetIngestor.class.getName()).log(Level.SEVERE, null, new AssetIngestorException("Cannot create asset: asset is null on path  " + assetPath));
+                } else {
```

Comment:
In this case report wouldn't be updated. Error will be logged only

Reply:
It could be very well the case that if the unit tests aren't asserting this behavior currently then the unit tests might be what needs adjusting.  I haven't looked to confirm this though.

Reply:
Agreed 100%.  However if the unit tests are failing because of the new error reporting then we might have to adjust them to assert this new behavior is correct, that's what I'm saying.